_posts/2020-05-13-2020-utah-lidar-acquisition.md

```diff
@@ -0,0 +1,68 @@
+---
+title: '2019 and 2020 Utah Aerial Lidar Acquisition'
+author:
+  display_name: Rick Kelson
+  email: rkelson@utah.gov
+date: 2020-05-13 12:10:00
+categories:
+- Featured
+tags:
+- lidar
+- 3dep
+---
+
+AGRC has awarded Aero-Graphics, Inc. and The Sanborn Map Company, Inc. to acquire [*20,958 square miles*](http://arcg.is/08W4bS "view project areas in ArcGIS Online") of aerial lidar elevation data. The lidar will be a combination of USGS Quality Level 1 (_QL1_), 8 points per square meter and Quality Level 2 (_QL2_), 2 points per square meter depending on the area. The 2019 QL2 project covering portions of Kane County, Utah was acquired last fall, is going through USGS Quality Assurance, and is scheduled to be completed by June 30, 2020. The 2020 acquisitions have begun and will continue throughout the year. These project areas will be completed in 2021 and project deliverables are scheduled to be completed by June 30, 2021.
```

Comment:
**nitpick**: if you want the links to be bold it's two `*`'s if you want it italic it's 1 `_`. Same for the link in the next paragraph.

Reply:
done